web_service/xmlrpc_test.py

```diff
@@ -0,0 +1,32 @@
+import xmlrpc.client
+
+server = "localhost"
+port = 8069
+db_name = "practice"
+user = "admin"
+password = "admin"
+api_key="c4c17454847c978f7ed8cc2a400c9b6eda3cd6ea"
```

Comment:
I don't see that you use this variable in the code. Please remove it if you don't need it.